src/String.elm

```diff
@@ -81,7 +81,7 @@ import Result exposing (Result)
 
 A `String` can represent any sequence of [unicode characters][u]. You can use
 the unicode escapes from `\u{0000}` to `\u{10FFFF}` to represent characters
-by their code point. You can also include the unicode characters directly.
```

Comment:
I believe that in this place you actually want to say "code point," since this is specifically referring to a Unicode code point (as opposed to an encoding of the point).